pkg/network/placement_configuration.go

```diff
@@ -0,0 +1,67 @@
+package network
+
+import (
+	"reflect"
+
+	cnao "github.com/kubevirt/cluster-network-addons-operator/pkg/apis/networkaddonsoperator/shared"
+	"github.com/pkg/errors"
+	corev1 "k8s.io/api/core/v1"
+)
+
+var defaultInfraNodeAffinity = corev1.NodeAffinity{
+	RequiredDuringSchedulingIgnoredDuringExecution: &corev1.NodeSelector{
+		NodeSelectorTerms: []corev1.NodeSelectorTerm{
+			getNodeSelectorTermForLabelExists("node-role.kubernetes.io/master"),
+		},
+	},
+}
+
+var defaultWorkloadNodeAffinity = corev1.NodeAffinity{
+	RequiredDuringSchedulingIgnoredDuringExecution: &corev1.NodeSelector{
+		NodeSelectorTerms: []corev1.NodeSelectorTerm{
+			getNodeSelectorTermForLabelExists("node-role.kubernetes.io/worker"),
```

Comment:
Unlike `master`, the worker label is not present on vanilla Kubernetes. We also can't make anti-affinity "do not deploy on masters", since that would prevent us from running on single-node clusters.

I'd say we should schedule worker deployments on all the nodes in the cluster, including masters. If somebody wants to limit them to workers, they can label workers accordingly and override the default.

Reply:
Aah I see. Using the defaults in the previous comment should fix this